internal/pkg/container/kernel.go

```diff
@@ -0,0 +1,47 @@
+package container
+
+import (
+	"path"
+	"path/filepath"
+	"sort"
+
+	"github.com/hpcng/warewulf/internal/pkg/wwlog"
+)
+
+var (
+	kernelSearchPaths = []string{
```

Comment:
If the kernel search paths are passed in as a parameter to KernelFind, it may be easier to write a functional test. The constants here are great. Consider parameterization in the function.

Reply:
Can you elaborate on this?